packages/mapbox/src/components/Map/__test__/map.spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Application } from 'react-rainbow-components';
+import Map from '../index';
+
+jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
+    GeolocateControl: jest.fn(),
+    Map: jest.fn(() => ({
+        addControl: jest.fn(),
+        on: jest.fn(),
+        remove: jest.fn(),
+        once: jest.fn(),
+    })),
+    NavigationControl: jest.fn(),
+}));
```

Comment:
we should add more test cases:
- should init map with the center passed (when pass a valid center new mapboxgl.Map should be called with the right one)
- should call navigator.geolocation.getCurrentPosition when center is not passed (here if we can mock the success and error callbacks we should test both cases, when new mapboxgl.Map is called with the resolved position and when called with the defaultCenter passed)